src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -237,6 +270,39 @@ export const WorkspaceDashboard = _.flow(
             storageCostEstimate
           ])
         ]),
+        div({ style: styles.header }, ['Tags']),
+        div({ style: { marginBottom: '0.5rem' } }, [
+          allTags && h(CreatableSelect, {
+            isClearable: true,
+            isSearchable: true,
+            value: newTag,
+            placeholder: 'Add a tag',
+            onChange: data => this.addTag(data.value),
+            styles: { container: base => ({ ...base, wordWrap: 'break-word' }) },
+            options: _.map(value => {
+              return {
+                value: value.tag,
+                label: `${value.tag} (${value.count})`
+              }
+            }, allTags)
+          })
+        ]),
+        div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+          _.map(tag => {
```

Comment:
If we're using a Select to pick tags, I would have expected that we'd be using it to display them as well, using its 'multiple' mode to create an integrated experience. Thoughts?

Reply:
Do you mean something like this?
<img width="338" alt="Screen Shot 2019-05-14 at 9 52 31 AM" src="https://user-images.githubusercontent.com/2395211/57703413-05d61480-762e-11e9-9cba-f4017a0ba6d3.png">

Reply:
Yes, that's what I was envisioning.